include/downloader.h

```diff
@@ -165,14 +165,16 @@ class Download {
  * A Downloader manages `Download` using aria2 in the background.
  * `Downloader` is threadsafe.
  * However, the returned `Download`s are NOT threadsafe.
+ * 
+ * @param sessionFileDir: The directory where aria2 will store its session file.
```

Comment:
This is the documentation of the `Downloader` class. You had to document this parameter in the constructor (which so far had no documentation since there was nothing to document).

Reply:
Yeah, I was not sure if the documentation of the class meant documentation for the class's constructor as well -- as if we assume one means the other. Fixing...